backend/functions/lib/use-cases/case-assignment.ts

```diff
@@ -79,9 +79,9 @@ export class CaseAssignmentUseCase {
           role: CamsRole[role],
           assignedOn: currentDate,
         },
-        context.session,
-        { updatedOn: currentDate },
+        context.session?.user,
       );
+      assignment.updatedOn = currentDate;
```

Comment:
Curious why we decided to update "updatedOn" after creating the audit record.  I see we did this also on line 127.  I must have missed the reasoning behind that.